test/test_manifest.json

```diff
@@ -5013,5 +5013,13 @@
       "rounds": 1,
       "type": "eq",
       "forms": true
+    },
+    {
+      "id": "issue12716",
+      "file": "pdfs/issue12716.pdf",
+      "md5": "b91ee4c41a5902710fcdd27166416fd6",
+      "rounds": 1,
+      "type": "eq",
```

Comment:
Unless actually necessary, please test only *one* page to save resources; adding `"lastPage": 1,` here will limit testing to just the *first* page.